ui/src/components/header/user-info-dialog/user-info-form/user-info-form.tsx

```diff
@@ -0,0 +1,132 @@
+import classNames from 'classnames'
+import { FormController } from 'components/form/form-controller'
+import { FormField } from 'components/form/form-field'
+import { FormConfig } from 'components/form/types'
+import { useUpdateUserInfo } from 'data-services/hooks/auth/useUpdateUserInfo'
+import { Button, ButtonTheme } from 'design-system/components/button/button'
+import { InputContent, InputValue } from 'design-system/components/input/input'
+import { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { STRING, translate } from 'utils/language'
+import { parseServerError } from 'utils/parseServerError/parseServerError'
+import { UserInfo } from 'utils/user/types'
+import { UserInfoImageUpload } from '../user-info-image-upload/user-info-image-upload'
+import styles from './user-info-form.module.scss'
+
+const IMAGE_MAX_SIZE = 1024 * 1024 // 1MB
+
+interface UserInfoFormValues {
+  name?: string
+  image?: File | null
+}
+
+const config: FormConfig = {
+  name: {
+    label: 'Name',
+  },
+  image: {
+    label: 'Image',
+    description:
+      'The image must smaller than 1MB. Valid formats are PNG, GIF and JPEG.',
```

Comment:
Can you use MAX_IMAGE_SIZE in this string?

Reply:
Yes good point!!